src/ledger/parse/account-trustline.ts

```diff
@@ -1,40 +1,24 @@
 import {parseQuality} from './utils'
 import {removeUndefined} from '../../common'
-
-type Trustline = {
-  account: string, limit: number, currency: string, quality_in: number|null,
-  quality_out: number|null, no_ripple: boolean, freeze: boolean,
-  authorized: boolean, limit_peer: string, no_ripple_peer: boolean,
-  freeze_peer: boolean, peer_authorized: boolean, balance: any
-}
-
-type TrustlineSpecification = {}
-type TrustlineCounterParty = {}
-type TrustlineState = {balance: number}
-type AccountTrustline = {
-  specification: TrustlineSpecification, counterparty: TrustlineCounterParty,
-  state: TrustlineState
-}
+import {Trustline, ParsedTrustline} from '../../common/types/objects/trustlines'
 
 // rippled 'account_lines' returns a different format for
 // trustlines than 'tx'
-function parseAccountTrustline(trustline: Trustline): AccountTrustline {
+function parseAccountTrustline(trustline: Trustline): ParsedTrustline {
   const specification = removeUndefined({
     limit: trustline.limit,
     currency: trustline.currency,
     counterparty: trustline.account,
     qualityIn: parseQuality(trustline.quality_in) || undefined,
     qualityOut: parseQuality(trustline.quality_out) || undefined,
     ripplingDisabled: trustline.no_ripple || undefined,
-    frozen: trustline.freeze || undefined,
-    authorized: trustline.authorized || undefined
+    frozen: trustline.freeze || undefined
   })
   // rippled doesn't provide the counterparty's qualities
   const counterparty = removeUndefined({
     limit: trustline.limit_peer,
     ripplingDisabled: trustline.no_ripple_peer || undefined,
-    frozen: trustline.freeze_peer || undefined,
-    authorized: trustline.peer_authorized || undefined
```

Comment:
I think we need to keep `authorized`. Unless there's a reason it was removed?

Reply:
I didn't see `peer_authorized` mentioned anywhere here, and assumed it was an old removed property / always undefined. https://ripple.com/build/rippled-apis/#account-lines

Happy to put it back in if that's not true (& create issue to document)

Reply:
I think it is missing from the documentation. I've opened a PR for the docs to add it in.

Reply:
👍 added back